src/amber/server/server.cr

```diff
@@ -43,9 +43,10 @@ module Amber
 
     def run
       thread_count = settings.process_count
+      env_hash = ENV.to_h
       if Cluster.master? && thread_count > 1
         while (thread_count > 0)
-          Cluster.fork ({"id" => thread_count.to_s})
+          Cluster.fork (env_hash.merge({"id" => thread_count.to_s}))
```

Comment:
Can we do this in the cluster class instead?

Reply:
And I think crystal classes has a dup/clone method see https://github.com/amberframework/amber/blob/3374da08814340e50c4deb47daeaa8cb785311d8/src/amber/server/cluster.cr#L5